src/GovUk.Education.ExploreEducationStatistics.Content.Api/Migrations/20190718135116_AddOrderFieldToRelease.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Api.Migrations
+{
+    public partial class AddOrderFieldToRelease : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                name: "Order",
+                table: "Releases",
+                nullable: false,
```

Comment:
Are we happy that all of the data in the dev database is set to 0? I assume that actually would be okay?

Reply:
yep - it'll be OK as will get added as zero & only one release at the mo.